File: TypeScript/8-kyu/Beginner Series #2 Clock.ts
export function past(h: number, m: number, s: number): number {
  const hoursInMilliseconds = h * 3600000; 
  const minutesInMilliseconds = m * 60000; 
  const secondsInMilliseconds = s * 1000; 
  
  return hoursInMilliseconds + minutesInMilliseconds + secondsInMilliseconds;
}

File: Go/8-kyu/Beginner Series #2 Clock.go
package kata

func Past(h, m, s int) int {
  hoursInMilliseconds := h * 3600000
  minutesInMilliseconds := m * 60000
  secondsInMilliseconds := s * 1000
  
  return hoursInMilliseconds + minutesInMilliseconds + secondsInMilliseconds
}
